Lib/tkinter/auto_complete.py

```diff
@@ -0,0 +1,155 @@
+try:
+    import tkinter as tk
+    from tkinter import ttk
+except ImportError:
+    # Python 2
+    import Tkinter as tk
+    import ttk
+
+__all__ = ["AutocompleteEntry"]
+
+NO_RESULTS_MESSAGE = "No results found for '{}'"
+
+
+class AutocompleteEntry(tk.Frame, object):
```

Comment:
```suggestion
class AutocompleteEntry(ttk.Frame):
```
We should prefer the ttk widgets